package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Locally run Ethereum client for developing the Streamr services",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint ./*.js"
```

Comment:
lint, good! We should add this to CI as well as an acceptance test.

Reply:
Do you know how to change the travis script? I didn't find a way.. Or who might know this?

Reply:
aight, let's see if my fix did it or not